bazel/README.md

```diff
@@ -365,7 +365,7 @@ The following optional features can be enabled on the Bazel build command-line:
   `--define log_debug_assert_in_release=enabled`. The default behavior is to compile debug assertions out of
   release builds so that the condition is not evaluated. This option has no effect in debug builds.
 * memory-debugging (scribbling over memory after allocation and before freeing) with
-  `--define tcmalloc=debug`.
+  `--define tcmalloc=debug`. Note this option cannot be used with FIPS-compliant mode BoringSSL.
```

Comment:
Can we catch this in the .bzl?

Reply:
Yeah I was trying to do this but didn't find a good way.
https://docs.bazel.build/versions/master/be/general.html#config_setting